package.json

```diff
@@ -7,8 +7,5 @@
     "url": "https://github.com/kiahjh/next-bg-image.git"
   },
   "homepage": "https://github.com/kiahjh/next-bg-image#readme",
-  "workspaces": [
-    "next-bg-image"
-  ],
```

Comment:
this wasn't helping, and was also cuasing me some weird issues with my `npub` script...